packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -353,32 +345,16 @@ class IOSDevice extends Device {
       return LaunchResult.failed();
     }
 
-    // Step 2.5: Generate a potential open port using the provided argument,
-    // or randomly with the package name as a seed. Intentionally choose
-    // ports within the ephemeral port range.
-    final int assumedObservatoryPort = debuggingOptions?.deviceVmServicePort
-      ?? math.Random(packageId.hashCode).nextInt(16383) + 49152;
-
     // Step 3: Attempt to install the application on the device.
     final String dartVmFlags = computeDartVmFlags(debuggingOptions);
     final List<String> launchArguments = <String>[
       '--enable-dart-profiling',
-      // These arguments are required to support the fallback connection strategy
-      // described in fallback_discovery.dart.
-      '--enable-service-port-fallback',
       '--disable-service-auth-codes',
-      '--observatory-port=$assumedObservatoryPort',
       if (debuggingOptions.disablePortPublication) '--disable-observatory-publication',
       if (debuggingOptions.startPaused) '--start-paused',
       if (dartVmFlags.isNotEmpty) '--dart-flags="$dartVmFlags"',
       if (debuggingOptions.useTestFonts) '--use-test-fonts',
-      // "--enable-checked-mode" and "--verify-entry-points" should always be
-      // passed when we launch debug build via "ios-deploy". However, we don't
-      // pass them if a certain environment variable is set to enable the
-      // "system_debug_ios" integration test in the CI, which simulates a
```

Comment:
RIP `system_debug_ios` I kept trying to resurrect that to test the iOS 14 launch from home screen but I couldn't get it to work.  Want to delete it?
\cc @christopherfujino re https://github.com/flutter/flutter/issues/43029

Reply:
I think it's already deleted. We can't launch from home going forward so it doesn't seem worth keeping this code aound

Reply:
So it is. #68107
I was hoping to use it to show when it's launched from the home screen it shows the nice warning explaining what happened.  But I couldn't get it to work.